### PR TITLE
Add Andrew

### PR DESCRIPTION
**Pull request by Spandamn**
Okay lets go

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -339,6 +339,60 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		shortDesc: "Refer to manual",
+		name: "SQUAD UP",
+		isNonstandard: "Custom",
+		gen: 8,
+		pp: 1,
+		priority: 4,
+		flags: {},
+		stallingMove: true,
+		volatileStatus: 'protect',
+		sideCondition: 'craftyshield',
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: ???

Comment (Spandamn) [reply data/mods/ssb/moves.ts]: Protected from status conditions

Comment (Spandamn) [reply data/mods/ssb/moves.ts]: Status moves

File: data/mods/ssb/abilities.ts [hunk 2]
```diff
@@ -254,6 +254,59 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+				if (!this.effectData.clones) return;
+				return this.chainModify(Math.pow(1.1, this.effectData.clones));
+			},
+			onModifySpe(spa, pokemon) {
+				if (!this.effectData.clones) return;
+				return this.chainModify(Math.pow(1.1, this.effectData.clones));
+			},
+			onHit(target, source, move) {
+				if (move.category === 'Status' || move.selfdestruct) return;
+				this.effectData.clones += 2;
```
Comment (ghost) [inline data/mods/ssb/abilities.ts:281]:
```suggestion
				if (pokemon.m.squadup) return;
				this.effectData.clones += 2;
```

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (ghost) [reply data/mods/ssb/moves.ts]: you just need protect tbh

File: data/mods/ssb/abilities.ts [hunk 3]
```diff
@@ -254,6 +254,59 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+		name: "NINJA SQUAD",
+		condition: {
+			onModifySpA(spa, pokemon) {
+				if (!this.effectData.clones) return;
+				return this.chainModify(Math.pow(1.1, this.effectData.clones));
+			},
+			onModifySpe(spa, pokemon) {
+				if (!this.effectData.clones) return;
+				return this.chainModify(Math.pow(1.1, this.effectData.clones));
+			},
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]:
```suggestion
			onModifySpA(spa, pokemon) {
				if (!this.effectData.clones) return;
				return this.chainModify(1 + (this.effectData.clones / 10));
			},
			onModifySpe(spa, pokemon) {
				if (!this.effectData.clones) return;
				return this.chainModify(1 + (this.effectData.clones / 10));
			},
```

Comment (xInstruct) [reply data/mods/ssb/abilities.ts:277]: that will give you not exactly the correct maths

Comment (ghost) [reply data/mods/ssb/abilities.ts]: what you had before was inaccurate in precision

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (Spandamn) [reply data/mods/ssb/moves.ts]: okay

**Pull request merged**